src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/hosts/host-form/host-form.component.html

```diff
@@ -14,19 +13,26 @@
     <div cdsModalContent>
       <!-- Hostname -->
       <div class="form-item">
-        <cds-text-label label="Hostname"
-                        for="hostname"
-                        cdRequiredField="Hostname"
+        <cds-text-label for="hostname"
                         [invalid]="!hostForm.controls.hostname.valid && hostForm.controls.hostname.dirty"
                         [invalidText]="hostnameError"
-                        i18n>Hostname
+                        i18n>Hostname (required)
+          <cd-helper>
+            To add multiple hosts at once, you can enter:
+            <ul cdsList>
+              <li cdsListItem>a comma-separated list of hostnames <b>(e.g.: example-01,example-02,example-03)</b>,</li>
```

Comment:
i18n?

Reply:
no need. its already being applied to `cds-text-label` and `cd-helper` is inside that one so its already covered.